docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,92 @@
+# Allowing Groups to be Based on and Synced With External LDAP Records
```

Comment:
> @deads2k If we are just requiring users to supply opaque filter strings for LDAP requests like is done for @liggitt's implementation of the login, why do we need to specifically support any one schema?

It's been a while, but I remember dealing with schemas where groups were first class objects that contained lists of users and I also remember dealing with schemas where groups were done as decorations on users.  There were multiple minor variations inside those categories.  I'd like to know the cases we're making easy.  Can we make both easy?  Do we need different information for the different cases?  Are there other cool (horrifying) things I haven't seen before that look common?